component/prometheus/exporter/mysql/mysql.go

```diff
@@ -144,6 +154,7 @@ func (a *Arguments) Convert() *mysqld_exporter.Config {
 		PerfSchemaEventsStatementsTextLimit:  a.PerfSchemaEventsStatements.TextLimit,
 		PerfSchemaFileInstancesFilter:        a.PerfSchemaFileInstances.Filter,
 		PerfSchemaFileInstancesRemovePrefix:  a.PerfSchemaFileInstances.RemovePrefix,
+		PerfSchemaMemoryEventsRemovePrefix:   a.PerfSchemaMemoryEvents.RemovePrefix,
```

Comment:
The opposite of this should be added to `converter/internal/staticconvert/internal/build/mysqld_exporter.go` `func toMysqldExporter`